frontend/src/scenes/sessions/SessionsView.tsx

```diff
@@ -211,7 +217,9 @@ export function SessionsView({ personIds, isPersonPage = false }: SessionsTableP
             </div>
 
             <ResizableTable
-                locale={{ emptyText: 'No Sessions on ' + dayjs(selectedDate).format('YYYY-MM-DD') }}
+                locale={{
+                    emptyText: selectedDate ? 'No Sessions on ' + selectedDate.format('YYYY-MM-DD') : 'No Sessions',
```

Comment:
Nit: Date format would look better like this (and in line with the new tooltips & legends)
```typescript
selectedDate.year() == dayjs().year() ? 'MMM D' : 'MMM D, YYYY'
```

Reply:
cool! we should probably `util` this some day